browser/src/control/jsdialog/Widget.ScrolledWindow.js

```diff
@@ -30,11 +30,29 @@
 
 /* global JSDialog */
 
+function _hasDrawingAreaInside(children) {
+	if (!children)
+		return false;
+
+	for (var i in children) {
+		if (children[i].type === 'drawingarea')
+			return true;
+		if (_hasDrawingAreaInside(children[i].children))
+			return true;
+	}
+
+	return false;
+}
+
```

Comment:
Thank you! I tried using the data.children but sometimes was getting nothing. I see the importance of the first check. I then tried to use content but I also see that it's much better to use the JSON object instead of the HTMLCollection item (data.children VS content)